README.md

```diff
@@ -173,29 +162,8 @@ import (
 )
 
 func main() {
-    directCfg := &wavefront.DirectConfiguration {
-        Server : "https://INSTANCE.wavefront.com", // your Wavefront instance URL
-        Token : "YOUR_API_TOKEN",                  // API token with direct ingestion permission
-
-        // Optional configuration properties. Default values should suffice for most use cases.
-        // override the defaults only if you wish to set higher values.
-
-        // max batch of data sent per flush interval. defaults to 10,000.
-        // recommended not to exceed 40,000.
-        BatchSize : 10000,
-
-        // size of internal buffer beyond which received data is dropped.
-        // helps with handling brief increases in data and buffering on errors.
-        // separate buffers are maintained per data type (metrics, spans and distributions)
-        // defaults to 500,000. higher values could use more memory.
-        MaxBufferSize : 500000,
-
-        // interval (in seconds) at which to flush data to Wavefront. defaults to 1 Second.
-        // together with batch size controls the max theoretical throughput of the sender.
-        FlushIntervalSeconds : 1,
-    }
-
-    sender, err := wavefront.NewDirectSender(directCfg)
+    // 11111111-2222-3333-4444-555555555555 is the API token
+    sender, err := wavefront.NewSender("https://11111111-2222-3333-4444-555555555555@surf.wavefront.com")
```

Comment:
I would add a comment about the ports, for symmetry with the Proxy comment about ports (so they don't presume ports 2878 and 30001 for DI). Something like:
`// For Direct Ingestion endpoints, by default all data is sent to port 80 or port 443 for unencrypted and encrypted connections, respectively.`

Reply:
Done.